packages/tool-cache/src/tool-cache.ts

```diff
@@ -200,7 +200,18 @@ export async function extractTar(
   }
 
   dest = dest || (await _createExtractFolder(dest))
-  const tarPath: string = await io.which('tar', true)
+  let tarPath: string
+  if (IS_WINDOWS) {
+    // Do not which tar on Windows due to incompatibilities between BSD tar and GNU tar on Windows.
+    // The version of tar at C:\Windows\System32\tar.exe is BSD tar and works properly with
+    // Windows-style paths. Whereas GNU tar requires forward slashes and also requires the flag
+    // `--force-local` due to the `:` in Windows paths. This presents an incompatibility because
+    // BSD tar does not support the flag `--force-local` (invalid flag, fails the process).
+    tarPath = path.join(process.env['windir'] as string, 'System32', 'tar.exe')
+    fs.statSync(tarPath)
```

Comment:
What's the purpose of `fs.StatSync(tarPath)`? To throw if tar doesn't exist?

Reply:
should probably use `existsSync` instead of this mechanism, if that's the case...

Reply:
yes, i'll imrpove the error message

Reply:
actually tool runner provides a good error message so i'll just remove this line